### PR TITLE
feat: Dark Theme

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
Screenshot (108)
Screenshot (109)

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
UI looks good with dark theme
PR is ready to merge

**Review by romabulani — APPROVED** [review]
- Liked the theme toggle mode
- Good to merge

**Review by mdarif6 — APPROVED** [review]
-UI looks good.
- Short and nice folder structure.
- Please check css , I saw colors with out variable names.

File: src/App.css [hunk 1]
```diff
@@ -13,6 +13,13 @@
   --nav-box-shadow: #eee;
   --sidebar-active: #262f44;
 }
+
+[data-theme="light"] {
```
Comment (vinc3nati) [inline src/App.css:17]: nice use of `data-` attribute

**Review by vinc3nati — APPROVED** [review]

**Pull request merged**